py/file.go

```diff
@@ -10,17 +10,126 @@
 package py
 
 import (
+	"io"
+	"io/ioutil"
 	"os"
 )
 
-var FileType = NewTypeX("file", `represents an open file`,
-	nil, nil)
-
 type File os.File
 
+var FileType = NewType("file", `represents an open file`)
```

Comment:
please document this exported variable.

actually.. do we need it to be exported, at this time?

Reply:
Sorry, I copied what one of the other module did and tried to follow the same pattern. Happy to update to a more gopheric style if one is defined.

But I agree that in the current implementation there are a lot of objects that are exported and maybe don't really need to be.